arrow-syntax/build.gradle

```diff
@@ -14,7 +14,7 @@ apply from: "$DOC_CONF"
 apply from: "$PUBLISH_CONF"
 
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$KOTLIN_VERSION"
```

Comment:
I think we shouldn't change it because of these comments: https://github.com/arrow-kt/arrow/pull/2065

Reply:
Good one, thanks!